FusionIIIT/templates/department/cse_dep.html

```diff
@@ -414,6 +414,19 @@ <h1><center> Welcome to CSE Department </center></h1>
         <div class="ui vertical segment">
             <div class="content">
                 <div class="description">
+                    <a class="active item" href="./All_Students/{{1111}}">
+                        <!--   <button style="border: none; float: left; display: block;" class="ui inverted right floated small red button" onclick="requestModal()"> -->
```

Comment:
Remove these type of comments from your PR File changes.

Reply:
Necessary changes have been made. Please check it.